test/rabbitmq_spec.rb

```diff
@@ -0,0 +1,86 @@
+#!/usr/bin/env ruby
+# frozen_string_literal: false
+
+#
+# rabbitmq
+#
+# DESCRIPTION:
+#   Tests for rabbitmq.rb
+#
+# OUTPUT:
+#
+# PLATFORMS:
+#
+# DEPENDENCIES:
+#
+# USAGE:
+#   bundle install
+#   rake spec
+#
+# NOTES:
+#
+# LICENSE:
+# Copyright 2018 Mike Murray <37150283+monkey670@users.noreply.github.com>
+#
+# Released under the same terms as Sensu (the MIT license); see LICENSE
+# for details.
+
+require_relative './spec_helper.rb'
+require_relative '../lib/sensu-plugins-rabbitmq/rabbitmq.rb'
+
+def queue_test
+  {
+    'name' => 'example-queue1'
+  }
+end
+
+class DummyClass
+end
+
+describe Sensu::Plugin::RabbitMQ do
+  before(:each) do
+    @options = {
+      host: 'localhost',
+      port: 55_672,
+      username: 'guest',
+      password: 'guest'
+    }
+    @dummy_class = DummyClass.new
+    @dummy_class.extend(Sensu::Plugin::RabbitMQ::Common)
+  end
+
+  it 'Acquire Rabbitmq info resuce' do
+    allow(@dummy_class).to receive(:warning)
+  end
+
+  it 'Acquire Rabbitmq info no config' do
```

Comment:
I would personally either use a comma between `info` and `no` are append `specified` but I will update that post merge.